app/controllers/internal/users_controller.rb

```diff
@@ -57,7 +57,7 @@ def banish
   def full_delete
     @user = User.find(params[:id])
     begin
-      Moderator::DeleteUser.call_deletion(admin: current_user, user: @user, user_params: user_params)
+      Moderator::DeleteUser.call(admin: current_user, user: @user, user_params: user_params)
```

Comment:
Like in a previous PR i decided to rename the interface method to `call`, so we get a bit more standardization around this.